components/datadog/agent/helm.go

```diff
@@ -21,11 +21,12 @@ const (
 )
 
 type HelmInstallationArgs struct {
-	KubeProvider  *kubernetes.Provider
-	Namespace     string
-	ValuesYAML    pulumi.AssetOrArchiveArrayInput
-	Fakeintake    *ddfakeintake.ConnectionExporter
-	DeployWindows bool
+	KubeProvider       *kubernetes.Provider
+	Namespace          string
+	ValuesYAML         pulumi.AssetOrArchiveArrayInput
+	Fakeintake         *ddfakeintake.ConnectionExporter
+	DeployWindows      bool
+	EnableOrchestrator bool
```

Comment:
We don't want to add all kind of possible options here. The `ValuesYAML` is here to allow setting the custom configuration you want.

Reply:
Ah! Thank you. I misunderstood how that worked